### PR TITLE
Changed backend code to allow users to edit their post after allotted time

### LINKED ISSUE #27: Allow users to edit their posts/messages even after 1 hour
Estimated effort: 5 hours
Dependencies among tasks (if any): n/a
Initial assignments for team members: Ben & Melody
Milestones: Sprint 2
Acceptance criteria: users are allowed to edit their messages even after an hour since they originally posted it

### PR DESCRIPTION
**Pull request by bencondemi**
Changed the backend code to give users the ability to edit their own posts even after 1 hour has passed. This could be useful because it could allow users to come back to their questions/answers with new updates/information without having to crowd the screen with multiple replies and/or increase the overall cognitive load of the page itself.

Resolves https://github.com/CMU-313/nodebb-f24-d-buggers/issues/27

### CODE REVIEW

File: userguide.md [hunk 1]
```diff
@@ -45,7 +45,9 @@ To install the project and the new features, follow these steps:
 2. Click on an existing topic, or create one if there is none
 3. Locate the three dots when hovering over the post & click on it
 4. Click on the Endorse button
-5. ...
+5. Click on an existing reply, or create one if there is none
```
Comment (shinyumh) [inline userguide.md:48]: i think the steps you added are a repeat of steps 2-4?

Comment (shinyumh) [reply userguide.md:48]: also i would probably put this on a separate PR

File: dump.rdb
Comment (shinyumh) [inline dump.rdb:1]: i don't think you need to commit this binary file?

**Review by shinyumh — COMMENTED** [review]
the files you changed regarding users editing posts looks good to me! i think you correctly removed all the code blocks that have to do with enforcing the post edit time limit. i would probably remove the `dump.rdb` and `userguide.md` from this specific PR before merging

File: dump.rdb
Comment (bencondemi) [reply dump.rdb:1]: git was bothering me about it as I was trying to change branches. it wouldn't let me pull or push without staging it

Comment (shinyumh) [reply dump.rdb:1]: ah ok, then feel free to merge & create another PR to fix the changes. thanks!

File: userguide.md (hunk 1 above)
Comment (shinyumh) [reply userguide.md:48]: sorry i didn't read the reply part. ignore this comment!

**Review by shinyumh — APPROVED** [review]

**Pull request merged**